pkgs/servers/mail/dovecot/default.nix

```diff
@@ -74,5 +74,8 @@ stdenv.mkDerivation rec {
     description = "Open source IMAP and POP3 email server written with security primarily in mind";
     maintainers = with stdenv.lib.maintainers; [ peti rickynils fpletz ];
     platforms = stdenv.lib.platforms.unix;
+    tests = [
+      (import ../../../../nixos/tests/opensmtpd.nix {})
```

Comment:
There should also be a link to `nixos/tests/dovecot.nix`.

Reply:
Indeed, I had only picked the `opensmtpd` test I had written as an example for the purpose of the RFC, all tests should be added to all relevant packages, I think so too :) There appear to be ~200 of them, though, so I'm not sure all of them could reasonably go into a single PR without risking to have bitrotten before even being completed?